python/paddle/fluid/tests/unittests/test_quantile.py

```diff
@@ -0,0 +1,161 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle
+
+
+def np_quantile_multi_q(x, q, axis=None, keepdims=False):
+    if not isinstance(q, (list, tuple)):
+        return np.quantile(x, q, axis=axis, keepdims=keepdims)
+    else:
+        output = []
+        for q_num in q:
+            output.append(np.quantile(x, q_num, axis=axis, keepdims=keepdims))
```

Comment:
np.quantile support q as array, call np.quantile once instead of loop?

Reply:
done, use np.quantile directly.